pootle/apps/virtualfolder/utils.py

```diff
@@ -195,6 +197,14 @@ class DirectoryVFDataTool(RelatedStoresDataTool):
     def context_name(self):
         return self.context.name
 
+    @property
+    def max_unit_revision(self):
+        # todo use through table for other stats?
```

Comment:
Please lets not add TODO notes referring to other code, instead better create GH issues. TODO notes re following code are acceptable, but this is not the case.